exonum/src/runtime/rust/runtime_api.rs

```diff
@@ -14,14 +14,17 @@
 
 //! Rust runtime specific API endpoints.
 
+use semver::Version;
```

Comment:
With upcoming `rust` runtime extraction in mind, it could make sense to provide `semver::Version` as a public re-export from core.